test/dynamo/test_modes.py

```diff
@@ -244,6 +245,21 @@ def fn(x):
 
         self.assertEqual(cnt.frame_count, 2)
 
+    def test_torch_function_all_disabled_api(self):
```

Comment:
this is only really testing eager torch_function behavior (and not dynamo), right? `test/test_overrides.py` might be a better place for it.